roles/join_nodes/templates/join_config.j2

```diff
@@ -1,4 +1,9 @@
+{%- if ansible_check_mode -%}
```

Comment:
Check-mode doesn't mean token doesn't exist, it's only missing when doing dry-run after token expire.
(On a stash of myself, I ended-up with  `{%- if valid_bootstrap_tokens is truthy -%}`

Reply:
But I realized that's ansible >= 2.10, syntax

Reply:
Changed a bit detection method as discussed, so now it reuse existing token when a valid one exist and use a dummy one else.